File: src/com/l2kt/gameserver/network/serverpackets/GetItem.java
package com.l2kt.gameserver.network.serverpackets;

import com.l2kt.gameserver.model.item.instance.ItemInstance;

/**
 * format ddddd
 */
public class GetItem extends L2GameServerPacket
{
	private final ItemInstance _item;
	private final int _playerId;
	
	public GetItem(ItemInstance item, int playerId)
	{
		_item = item;
		_playerId = playerId;
	}
	
	@Override
	protected final void writeImpl()
	{
		writeC(0x0d);
		writeD(_playerId);
		writeD(_item.getObjectId());
		
		writeD(_item.getX());
		writeD(_item.getY());
		writeD(_item.getZ());
	}
}

File: src/com/l2kt/gameserver/network/serverpackets/GetItem.kt
package com.l2kt.gameserver.network.serverpackets

import com.l2kt.gameserver.model.item.instance.ItemInstance

/**
 * format ddddd
 */
class GetItem(private val _item: ItemInstance, private val _playerId: Int) : L2GameServerPacket() {

    override fun writeImpl() {
        writeC(0x0d)
        writeD(_playerId)
        writeD(_item.objectId)

        writeD(_item.x)
        writeD(_item.y)
        writeD(_item.z)
    }
}